packages/rancher-istio-1.7/charts/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
@brendarearden let's drop the first commit and refactor the second commit to remove 1.7 changes since 1.7 is no longer being released